ihmc-interfaces/src/main/messages/ros1/behavior_msgs/msg/ConditionNodeDefinitionMessage.msg

```diff
@@ -1,14 +1,23 @@
 # COUNTER TYPE
 # LLM TYPE
+# PROXIMITY TYPE
 
 int8 COUNTER_TYPE=0
 
 int8 LLM_TYPE=1
 
+int8 PROXIMITY_TYPE=2
+
+int8 XYZ_TYPE=0
+
+int8 XY_TYPE=1
+
+int8 Z_TYPE=2
```

Comment:
These constants should be prefixed with `PROXIMITY_` , you can remove the `_TYPE` suffix on these